test/index.test.js

```diff
@@ -344,4 +344,34 @@ if (ciJob() === 1) {
     expect(stderr).toEqual('')
     expect(stdout).toEqual('esm d\nesm a\nesm b\nesm browser c\nesm lib\n')
   })
+
+  it('copy package.json fields as a conditions for exports field', async () => {
+    let [normalizeCss] = await copyDirs('normalize-css')
+    await processDir(normalizeCss)
+
+    let packageJSON = await readFile(join(normalizeCss, 'package.json'))
+
+    expect(packageJSON.toString()).toMatchInlineSnapshot(`
+      "{
+        \\"name\\": \\"normalize-css\\",
```

Comment:
Can we `'` string to avoid `\"`?

Reply:
We can improve code readability by using `expect(packageJSON).toEqual(object)`  instead of using strings.